kerrokantasi/settings/base.py

```diff
@@ -1,12 +1,30 @@
 import os
+
+import environ
+import raven
```

Comment:
Soooo, why exactly _is_ this imported?

Reply:
Missing (WIP) tag from the PR. Ie. this is still WIP.